core/state/journal.go

```diff
@@ -90,8 +90,14 @@ type (
 		account *common.Address
 	}
 	resetObjectChange struct {
-		prev         *stateObject
-		prevdestruct bool
+		account                *common.Address
+		prev                   *stateObject
+		prevdestruct           bool
+		prevAccount            []byte
+		prevStorage            map[common.Hash][]byte
+		prevAccountOriginExist bool
```

Comment:
Could u split space here between Account & AccountOrigin and put some small documentation here for explaining the diferrent tracking here.

Reply:
Added